pkg/services/object/acl/v2/util_test.go

```diff
@@ -43,38 +48,48 @@ func testGenerateMetaHeader(depth uint32, b *acl.BearerToken, s *session.Token)
 
 func TestIsVerbCompatible(t *testing.T) {
 	// Source: https://nspcc.ru/upload/neofs-spec-latest.pdf#page=28
-	table := map[eacl.Operation][]eacl.Operation{
-		eacl.OperationPut:       {eacl.OperationPut},
-		eacl.OperationDelete:    {eacl.OperationPut, eacl.OperationHead, eacl.OperationSearch},
-		eacl.OperationHead:      {eacl.OperationHead},
-		eacl.OperationRange:     {eacl.OperationRange, eacl.OperationHead},
-		eacl.OperationRangeHash: {eacl.OperationRange, eacl.OperationHead},
-		eacl.OperationGet:       {eacl.OperationGet, eacl.OperationHead},
-		eacl.OperationSearch:    {eacl.OperationSearch},
+	table := map[eacl.Operation][]sessionSDK.ObjectVerb{
+		eacl.OperationPut:    {sessionSDK.VerbObjectPut, sessionSDK.VerbObjectDelete},
+		eacl.OperationDelete: {sessionSDK.VerbObjectDelete},
+		eacl.OperationGet:    {sessionSDK.VerbObjectGet},
+		eacl.OperationHead: {
+			sessionSDK.VerbObjectHead,
+			sessionSDK.VerbObjectGet,
+			sessionSDK.VerbObjectDelete,
+			sessionSDK.VerbObjectRange,
+			sessionSDK.VerbObjectRangeHash,
+		},
+		eacl.OperationRange:     {sessionSDK.VerbObjectRange, sessionSDK.VerbObjectRangeHash},
+		eacl.OperationRangeHash: {sessionSDK.VerbObjectRangeHash},
+		eacl.OperationSearch:    {sessionSDK.VerbObjectSearch, sessionSDK.VerbObjectDelete},
```

Comment:
It doesn't match the spec now.

Reply:
`Search` op can be executed within `Search` and `Delete` session, what mismatches?

Reply:
Ok, I see, you swapped the order